stakepooldclient/stakepooldclient.go

```diff
@@ -341,12 +341,9 @@ func (s *StakepooldManager) syncScripts(multiSigScripts []models.User) error {
 
 	log.Info("syncScripts: Attempting to synchronise redeem scripts across voting wallets")
 
-	// Fetch the redeem scripts from each server.
-	redeemScriptsPerServer := make([]map[chainhash.Hash]*ScriptHeight,
-		len(s.grpcConnections))
+	// Get all scripts from db
 	allRedeemScripts := make(map[chainhash.Hash]*ScriptHeight)
 
-	// add all scripts from db
```

Comment:
on line 350, there is a log that says "Skipping script", but I feel like that implies it will continue when really it returns there. I think "failed on" is more accurate. And we don't need capitalization, I believe. Again, I know this isn't your code, just while you're here.